taca/analysis/analysis_nanopore.py

```diff
@@ -372,41 +355,9 @@ def ont_updatedb(ont_run):
         # if the run document is marked as "finished"
         if sesh.check_run_status(ont_run) == "finished":
             logger.info(
-                f"Run {ont_run.run_id} exists in the database as an finished run."
+                f"Run {ont_run.run_id} exists in the database as an finished run, do nothing."
             )
 
-            glob_html = glob.glob(ont_run.run_dir + "/report*.html")
-            if len(glob_html) == 0:
-                error_message = f"Run {ont_run.run_id} is marked as finished, but missing .html report file."
-                logger.error(error_message)
-                raise AssertionError(error_message)
-            elif len(glob_html) > 1:
-                error_message = f"Run {ont_run.run_id} is marked as finished, but contains conflicting .html report files."
-                logger.error(error_message)
-                raise AssertionError(error_message)
-
-            logger.info(f"Transferring the run report to ngi-internal.")
-
-            # Transfer the MinKNOW .html report file to ngi-internal, renaming it to the full run ID. Requires password-free SSH access.
-            report_dest_path = os.path.join(
-                CONFIG["nanopore_analysis"]["ont_transfer"]["minknow_reports_dir"],
-                f"report_{ont_run.run_id}.html",
-            )
-            transfer_object = RsyncAgent(
-                glob_html[0],
-                dest_path=report_dest_path,
-                validate=False,
-            )
-            try:
-                transfer_object.transfer()
-                logger.info(
-                    f"Successfully transferred the MinKNOW report of run {ont_run.run_id}"
-                )
-            except RsyncError:
-                msg = f"An error occurred while attempting to transfer the report {glob_html[0]} to {report_dest_path}"
-                logger.error(msg)
-                raise RsyncError(msg)
-        logger.info(f"Database update for run {ont_run.run_id} successful")
     except Exception as e:
```

Comment:
This try/except is really long. Might be worth considering splitting it up into several smaller ones.

Reply:
1481d66cfc90ceac0010f0ba6c71bfa89bd24935